samples/hello-oboe/src/main/cpp/PlayAudioEngine.cpp

```diff
@@ -112,25 +83,7 @@ void PlayAudioEngine::createPlaybackStream() {
     }
 }
 
-/**
- * Sets the stream parameters which are specific to playback, including device id and the
- * callback class, which must be set for low latency playback.
- * @param builder The playback stream builder
- */
-void PlayAudioEngine::setupPlaybackStreamParameters(oboe::AudioStreamBuilder *builder) {
-    builder->setAudioApi(mAudioApi);
```

Comment:
Did we lose the ability to set the AudioAPI?

Reply:
We can still set the AudioAPI in  `PlayAudioEngine::setAudioApi`. This method is called when the user modifies the AudioAPI spinner in the interface. When the stream starts, it uses the default.